src/hyperion/experiment_plans/robot_load_then_centre_plan.py

```diff
@@ -168,6 +169,9 @@ def robot_load_and_energy_change(
 
     yield from bps.wait("robot_load")
 
+    yield from bps.abs_set(composite.thawer.thaw_for_time_s, thawing_time)
```

Comment:
this should have a group

Reply:
Group is added in https://github.com/DiamondLightSource/hyperion/pull/1527 I will merge the two PRs